src/unity/paths/operator_orgs_orgId.yml

```diff
@@ -0,0 +1,36 @@
+get:
+  operationId: getOrganizationById
+  tags:
+    - Organizations
+    - Operator
+  summary: Get organization by id
+  parameters:
+    - $ref: "../../common/parameters/TraceSpan.yml"
+    - in: path
+      name: orgId
+      schema:
+        type: string
+      required: true
+      description: the idpe id of the org
+  responses:
+    '200':
+      description: Organization
+      content:
+        application/json:
+          schema:
+            allOf:
+              - $ref: '../schemas/Organization.yml'
+              - properties:
```

Comment:
get org returns an org with its account nested

Reply:
I think this is related to the concern above where an account has an org property and this will be endless